cms/djangoapps/contentstore/courseware_index.py

```diff
@@ -628,6 +633,7 @@ def index_about_information(cls, modulestore, course):
                     course_info['content'][about_information.property_name] = analyse_content
                 if about_information.index_flags & AboutInfo.PROPERTY:
                     course_info[about_information.property_name] = section_content
+        course_info['category'] = map(lambda x: x.get_root().name, CourseCategory.objects.filter(courses__id=course.id))
```

Comment:
I agree that list comprehension is better than `map` and `lambda` here. List works faster than map. I've pushed a fix.

About `get_root`. I dunno, because of our colleges from the Duck team make a decision to use tree package for categories. I don't see a concern here. The category is a nested model, so tree to get better visualization for admin panel and convenience when working with a similar model. I personally do not have objections here.